Outreachy internship - OSCSA subpage contribution

codeinvoke7 opened the pull request:
I worked on the open educational resources bullet point

I embedded the youtube video in an iframe.
I created the same page in Arabic.
I made sure I attached the header and the footer both in the English and Arabic page.

HOW I CAN IMPROVE THE PAGE MORE
Adding a button that will lead to the subpage will improve user experience rather than clicking on the text
Each subpage should have not only the header and footer but also the contact form

codeinvoke7 commented:
Hi @BatoolMM I'm an outreachy intern applicant and I just made my contribution please do review and give feedback, thanks

BatoolMM commented:
Thank you @codeinvoke7 for your work. A few comments about the PR.
- The content is very brief, you need to elaborate more - similar to this PR #38.
- I love how you managed to get the right alignment for Arabic. The 1st one I saw so far!
- Add where the text is cited from.
- Add where the image is taken from
- Add a few sentences about the video. 
> This video is part of [Open Life Sciences Programe](https://openlifesci.org/) and the subtitles are available in Arabic and English. The OLS program is for people interested in applying open principles in their work and becoming Open Science ambassadors in their communities.
You may also add a reference for the video:
> Please cite: Malvika Sharan, Emmy Tsang, Yo Yehudi, Bérénice Batut, Carlos Martinez, Helena Rasche, Hao Ye, Esther Plomp, Andre Maia Chagas, Georgia Aitkenhead, Suzanne lwai, Otis Smith, Emma Anne Harris, Thomas Mboa, Alex Holinski, Zulidyana Rusnalasari, Nelsy Mtsweni, Rowland Mosbergen, Iratxe Puebla, … Anelda van der Walt. (2021, September 19). Mini Talks/Videos about Open Science from OLS-3 Cohort (v.1.1). Zenodo. https://doi.org/10.5281/zenodo.5515763
- Since open educational resources have been submitted with multiple PR. Would you like to try "Citizen Science". You can re-use materials from [the Turing Way here](https://the-turing-way.netlify.app/reproducible-research/open/open-scholarship.html?highlight=citizen%20science) with cittaion.

Thank you again for your hard work and looking for more contributions!

codeinvoke7 commented:
Thank you @BatoolMM  for your review, I'll go ahead and address the issues you mentioned. 

Yes I will like to work on the "citizen science". 

On citizen science for more clarity what do i need to do exactly should I add it as a bullet point then link it to a subpage with the full write 'the Turing way'

codeinvoke7 commented:
Hello @BatoolMM my mentor Thank you for your comments. I have made the corrections.
I added more text for more elaboration
I added where the text is cited from.
I added an image and where the image is taken from.
I added a few sentences about the video.
All were updated both in English and Arabic
Please do review again and give feedback Thank you so much

And for the citizen science please I will like you to explain more on how I can go about it thanks

codeinvoke7 commented:
Hello my mentor @BatoolMM I want to call your attention again to examine my contribution based on your comments

alswajiab commented:
Thank you @codeinvoke7 for your work. I have some comments:
- [ ] Have a way to get back to the main page.
- [ ] Make the [image](https://deploy-preview-37--oscksa.netlify.app/image/5R_OER.png) resize dynamically when the window browser's width changes.
- [ ] Match the listing of subheadings of the subpage same as the main page (adding check marks).
- [ ] Add Video citation as a caption (not as a separate text).
- [ ] - [ ] Add simple description instead of listening URLs for (Resources) section.
- [ ] Fix the Arabic language button not directing to the Arabic page.

codeinvoke7 commented:
Hello @BatoolMM and @alswajiab I have done the corrections as specified which includes:
I added a way to get back to the previous page from the subpage by clicking the 'Home' navigation
I made the image to resize dynamically
I matched the listing of the subpage subheadings to use by adding check mark
Video caption added
A simple description on the resource section
I fixed the Arabic button to not direct to the Arabic page

 Please do check and give feedback thanks very much

codeinvoke7 commented:
Hello my mentor @BatoolMM I also fixed the hamburger on the Arabic page mobile view which was not showing before

alswajiab commented:
Thank you @codeinvoke7 for your contribution, everything looks good to me but please provide a way to the home page like :
`Home / OPEN EDUCATIONAL RESOURCES`

codeinvoke7 commented:
@alswajiab Thanks again for the review, guidelines and corrections. I have added a way to go back pls review once more I await your feedback. Thank you
